hackclub.com.yaml

```
@@ -2577,7 +2577,7 @@ whs:
 wl:
   - ttl: 1
```

```suggestion
   - ttl: 600
```

Just taking this opportunity to correct DNS TTLs here :)